Cargo.toml

```diff
@@ -2,8 +2,9 @@
 name        = "nix"
 description = "Rust friendly bindings to *nix APIs"
 edition     = "2021"
+resolver    = "2"
```

Comment:
It seems that [this is already enabled since we are using Rust 2021](https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html)?